addon/templates/components/frost-modal-about-dialog.hbs

```diff
@@ -4,8 +4,8 @@
 >
   {{frost-icon
     class='frost-modal-about-dialog-close-icon'
-    icon='about-close'
-    pack='frost-modal'
+    icon='round-close'
+    pack='frost'
```

Comment:
Should we remove pack here since it's a default? I don't know what's the convention we go, but I saw the same usage in frost-sort without specifying 'frost' pack